.travis.yml

```diff
@@ -1,10 +1,53 @@
-language: python
-python: 2.7
-env:
-  - TOXENV=flake8
-install:
-  - pip install tox
-script:
-  - tox
-notifications:
-  email: kpennington@mozilla.com
+cache: pip
+
+sudo: required
+jobs:
+  include:
+    - stage:
+      language: python
+      python: 2.7.12
+      env:
+        - MOZ_HEADLESS=1
+      addons:
+        firefox: latest-nightly
+      env: TOXENV=py27
```

Comment:
You have 2 ```env``` properties here. I don't think headless is being set.

Reply:
a-ha. Thanks for that catch